### DISCUSSION TITLE
Please add mp4 and flv support for video

### DISCUSSION THREAD

**Discussion by maxlinux2000** [openbroadcaster/observer#26] [Ideas]
Actually I can use a simple raspberryPi2 like as media player for streaming HD mp4 videos at 25fps with a 10-15% of cpu with the original quality of the videos.
This is possible using ffmpeg with "-c:v copy -c:a copy" and videos in mp4 codified in HD 1280x720 h264 800kbs + aac 128kbs.
This system have a superb quality and I can streaming in my server using nginx+rtsp as in youtube and twitch at same time  (i'm using the youtube like opensource project called now avideo and youphptube before).
But... hot to do the same with ogg format without recode the stream en real time?

regards
MaX

**Comment by radiorob** [comment]
Already supports MP4, AVI, WMV, MOV and Ogg Theora.  Adding FLV is possible, if there is enough demand

**Comment by peterpetrov** [comment]
Hey @radiorob, I just installed the server for testing purposes. I can see that mp4 is supported on audio only and when i try to upload mp4 video file it says that the media is not supported. Does this makes sense? 

Peter

**Comment by radiorob** [comment]
did you follow the instructions in README?

http://support.openbroadcaster.com/Troubleshooting#post-installation-server-troubleshooting

What ver and OS? Source of your mp4 video, encoder?

Rob

**Comment by Neepawa** [comment]
> Hey @radiorob, I just installed the server for testing purposes. I can see that mp4 is supported on audio only and when i try to upload mp4 video file it says that the media is not supported. Does this makes sense?
> 
> Peter

To upload MP4 files, you need to go into **Admin > Media Settings** and select **MOV (Apple Quicktime).** The reason is a library OpenBroadcaster uses to identify files (`extras/getid3`) returns  *Quicktime/MOV* as the file type for MP4 files.
